celery/worker/control.py

```diff
@@ -515,7 +516,8 @@ def autoscale(state, max=None, min=None):
 def shutdown(state, msg='Got shutdown from remote', **kwargs):
     """Shutdown worker(s)."""
     logger.warning(msg)
-    raise WorkerShutdown(msg)
+    logger.critical(msg)
```

Comment:
We already provide the message as a warning here. Don't change existing behavior.